src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -105,12 +106,10 @@ public static bool IsScreenReaderEnabled
         /// <returns></returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static bool Say(string sentence, bool discardable)
+        public static void Say(string sentence, bool discardable)
```

Comment:
Oh.. the return type is changed.
AFAIK, this feature was handled by Mr.Song, so I don't know the history.

@dongsug-song , Could you check this feature?
1. Are there any reason to add `Say` and other features previously?
2. Is it okay to change the return type? + VD / DA team doesn't use these features anymore?

Reply:
The problem with the return type is that the interop (`Interop.Accessibility.Say`) was declared in C# as having a `bool` return type, but it returned `void` from C++, so the value would be indeterminate (meaningless).

Therefore I also changed the public API from `bool` to `void`, but I can keep `bool` and just add `return true` at the end if needed for compatibility with existing code. (But I doubt that anyone used the return value since it was basically meaningless.)

Reply:
@Seoyeon2Kim 
I think Mr. aswigon's opinion is reasonable. 
as you pointed out, it had been added for GBM, but actually it is not used currenly as far as I know, and it is hidden API so that it can be changed without ACR. 
thank you for your careful comments but I think it would be OK to follow Mr. aswigon's idea.

Reply:
Understood.
As aswigon mentioned, the return value was meaningless before.
Also, this hidden API is not used anywhere now, so it can be changed. 
Thank you for checking it once. :)